subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/TestingIntegrationTest.groovy

```diff
@@ -276,13 +277,13 @@ class TestingIntegrationTest extends JUnitMultiVersionIntegrationSpec {
 
         when:
         executer
-                .withArgument("-Porg.gradle.java.installations.paths=${AvailableJavaHomes.getAvailableJvms().collect { it.javaHome.absolutePath }.join(",")}")
-                .withToolchainDetectionEnabled()
+            .withArgument("-Porg.gradle.java.installations.paths=${AvailableJavaHomes.getAvailableJvms().collect { it.javaHome.absolutePath }.join(",")}")
+            .withToolchainDetectionEnabled()
         buildScript """
             plugins {
                 id("java")
             }
-            ${withJava11Toolchain()}
+            ${javaPluginToolchainVersion(11)}
```

Comment:
Do you know why this test requires Java 11 explicitly and not any JVM?

Reply:
I don't know. The setup for the test contains an extensive commentary about specific dependencies. Given that this part of the refactor was already outside of the previous changes, I decided to not modify the behavior of the test.